x-pack/legacy/plugins/maps/public/connected_components/widget_overlay/layer_control/layer_toc/toc_entry/_toc_entry.scss

```diff
@@ -41,6 +41,11 @@
   pointer-events: none !important;
 }
 
+.mapTocEntry-isSelected {
+  background-color: $euiColorLightShade;
```

Comment:
I've been playing around with colors and I suggest the following change:
```suggestion
  background-color: tintOrShade($euiColorLightShade, 60%, 20%);
```

<img width="1636" alt="maps@2x" src="https://user-images.githubusercontent.com/2750668/77759963-870f7d80-702d-11ea-9465-8e5171915e61.png">